cpp/include/cudf/hashing.hpp

```diff
@@ -144,6 +145,24 @@ std::unique_ptr<column> md5(
   rmm::cuda_stream_view stream        = cudf::get_default_stream(),
   rmm::mr::device_memory_resource* mr = rmm::mr::get_current_device_resource());
 
+/**
+ * @brief Computes the hash value of each row in the input set of columns
```

Comment:
please add detail about the hash algorithm used.

Reply:
Like what?

Reply:
I think he means that the brief does not mention the name of the algorithm. However this is true for all the hash functions in libcudf currently. I would defer this change to #13706 where it can be done uniformly for all hash functions.

Reply:
I noted this in #13706. I think this can be resolved.

Reply:
Sounds good.